server/README.md

```diff
@@ -61,9 +61,13 @@ The server executable can be built with `nix build .#cardano-trasaction-lib-serv
     - `application/json`
 
 - Bytes to hash as hexadecimal string (`application/json;charset=utf-8`, `application/json`):
+- Hashing method from PlutusTx to hash with; one of `Sha2_256`, `Sha3_256`, `Blake2b_256`
 
 ```javascript
-"666f6f"
+{
+  "bytesToHash" : "68656C6C6F20776F726C64"
```

Comment:
I think the field names are a bit verbose. Why not just `bytes` and `method`?

Reply:
I was expecting naming clashes when not using specific names, and indeed, `method` from `BytesToHash` would conflict with `method` from `HashBytesRequest`. I could rename it by writing custom `FromJSON` instances, do you want me to do so?

Reply:
Do you mean record name clashes? You could also just use `DuplicateRecordFields`, I have no problem with that (and in fact would be my first choice)